src/Stripe.android.js

```diff
@@ -21,6 +21,8 @@ class Stripe {
     StripeModule.deviceSupportsAndroidPay()
   )
 
+  setAndroidPayMode = StripeModule.setAndroidPayMode
```

Comment:
```js
setAndroidPayMode = mode => (
  StripeModule.setAndroidPayMode(mode)
)
```

Reply:
it's unused now so i removed that method.